app/controllers/reviews_controller.rb

```diff
@@ -0,0 +1,25 @@
+class ReviewsController < ApplicationController
+
+  before_action :require_user
+
+  def create
+    @video = Video.find(params[:video_id])
+    review = Review.new review_params.merge!(video_id: params[:video_id], user: current_user)
```

Comment:
I would certainly change this part `video_id: params[:video_id]` to `video: @video` or `video_id: @video.id`.

Reason for this is - user has some control over params and he can manipulate it in browser. You allow him to manipulate it some via permitting strong params (when it comes to writing to database). Also in previous line you are finding the @video - so if this is not going to be successful, then review is not going to be created.

Another reason that comes to my mind could be if you had a private videos and user could pass in an id of a video that is not allowed to him. Basically you could do something like `@video = Video.public.find(params[:video_id])` and have it filtered there first instead of filtering it also in review params hash. I hope you get my point.